R/draws.R

```diff
@@ -68,6 +68,10 @@
 #' Note that LOCF is generally not a principled imputation method and should only be used when appropriate 
 #' for the specific covariate.
 #'
+#' Please note that there is no special provisioning for "baseline". If you do not want baseline
+#' observations to be included in the model then these should be removed in advance from `data`. 
+#' The imputation model(s) created by `draws()` will fit the distribution to all levels of `data["visit"]`.
```

Comment:
The imputation model(s) created by `draws()` will fit the distribution to all levels of `data["visit"]`. I think the above is clear enough and we can remove this sentence?